lib/server.js

```diff
@@ -135,10 +135,15 @@ module.exports = coroutine(function*(req, res, flags, current, ignoredFiles) {
       res.end()
       return
     }
-
-    let indexPath = path.join(related, '/index.html')
+    
+    const { entry } = flags
+    let indexPath = path.join(related, '/' + entry)
     res.setHeader('Content-Type', mime.contentType(path.extname(indexPath)))
```

Comment:
Shouldn't this `setHeader` come after `indexPath` is definitively determined? If an invalid/non-existent entry file is specified, the logic is that it will default to index.html, but `setHeader` will have already been called.

Reply:
Yep, but in current code without my feature, exists the same problem, `setHeader` called only once and if `index.html` will not exists, that will error too, there are no check for not existing `index.html`

 I am just want to add the feature and use it, bug fixing will be a next steps😄

Reply:
good call 👍

Reply:
sorry if I'm wrong😅